litmus/director/TCID-DIR-OP-INSTALL-OPENEBS/test.yml

```diff
@@ -262,6 +262,17 @@
           retries: 50
           delay: 5
 
+        - name: Fetch nodes with control plane labels
+          shell: kubectl get nodes --show-labels --no-headers | grep {{ control_plane_node_label }} | wc -l
+          register: labelled_node_count
+
+        - name: Verify number of daemonset pods equal to labelled nodes
+          shell: kubectl get pods -n {{ namespace }} | grep {{ item }} | grep -v operator | awk '{print $3}' | grep Running | wc -l
+          register: ds_pods_count
+          with_items:
+            - "{{ openebs_ds_components }}"
```

Comment:
why do we have loop here @shiveshabhishek

Reply:
and one more thing. This will error out when the pod is in containercreating state. We need to employ retries with several iteration

Reply:
> why do we have loop here @shiveshabhishek

As of now, we have single component on daemonsets. If we'll have multiple components, we can directly add those to  `openebs_ds_components`